### ISSUE TITLE
stock constraint_offset(low, high, seed, offset) code unreachable solution

### ISSUE THREAD

**Issue by djearthquake** [alliedmodders/amxmodx#690]
```PHP
/// 2019 Patch for stock per return 0 angers compiler.
/// Set_Roll was used in this illustration. Feel free to take it!
/// high = 23, low = 0, offset = -23 Military time example.
/// Otherwise just comment out the return 0 to use FCN.

/**
 * Computes an offset from a given value while constraining it between the
 * specified bounds, rolling over if necessary.
 * 
 * @note Example: The range is 1-5 and the base value (seed) is 3, the offset
 *       that the value should be moved by is also 3. Offsetting the value by 3
 *       would result in 6, but it is to be constrained between 1 and 5. With
 *       clamp() this would result in 5, but this function rolls the value over
 *       and returns 1 instead.
 *           
 * @param low       Lower bound
 * @param high      Higher bound
 * @param seed      Base value
 * @param offset    Offset to move
 *
 * @return          Computed offset value between specified bounds
 */

/*stock constraint_offset(low, high, seed, offset)
{
        new numElements = high - low + 1;
        offset += seed - low;

        if (offset >= 0)
        {
                return low + (offset % numElements);
        }
        else
        {
                return high - (abs(offset) % numElements) + 1;
        }

///     return 0;       // Makes the compiler happy -_-
}*/



stock Set_Roll(low, high, seed, offset)

{
        {
        new numElements = high - low + 1;
        offset += seed - low;

        if (offset >= 0)
        {
                return low + (offset % numElements);
        }
        else
        {
                return high - (abs(offset) % numElements) + 1;
        }
        }
}
````

**Comment by Arkshine** [comment]
Could you format properly your code please?

When pasting block of code, select all the lines and use the code button:

![image](https://user-images.githubusercontent.com/360640/54349659-08c38180-464c-11e9-9318-edfa2c29294d.png)

**Comment by djearthquake** [comment]
How careless. Thank you.  Sorry about that. Taken care of.

**Comment by djearthquake** [comment]
This should be linked to https://github.com/alliedmodders/amxmodx/pull/784 from https://github.com/alliedmodders/amxmodx/issues/782 months later after I already found this. I could have been assigned the task. This was an easy one.

**Comment by djearthquake** [comment]
Fixed by [#784](https://github.com/alliedmodders/amxmodx/pull/784).
